projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts

```diff
@@ -245,7 +245,8 @@ export function loginCustomerInStorefront(customer) {
 export function agentSignOut() {
   const tokenRevocationAlias = loginHelper.listenForTokenRevocationRequest();
   cy.get('button[title="Sign Out"]').click();
-  cy.wait(tokenRevocationAlias).its('response.statusCode').should('eq', 200);
+  // due to flaky not to check 200 status because revocation get called twice and some time one of call fails
```

Comment:
`// When the agent signs out, there are two revocations made simultaneously - the second one occasionally fails, though it is not necessary. We therefore are not interested in the status code of the second one.`